The <sys/types.h> header is apparently not automatically pulled in
by including <stdio.h> on Tiger, resulting in off_t being undefined

see ticket #63393
kencu@macports.org

diff --git src/gdbm.h.in.orig src/gdbm.h.in
index be6a11a..b1aaabc 100644
--- src/gdbm.h.in.orig
+++ src/gdbm.h.in
@@ -30,6 +30,7 @@
 # define _GDBM_H_
 
 # include <stdio.h>
+# include <sys/types.h>
 
 /* GDBM C++ support */
 # if defined(__cplusplus) || defined(c_plusplus)
